brili.ts

```diff
@@ -762,7 +762,7 @@ function evalInstr(instr: bril.Instruction, state: State): Action {
 
   case "int2char": {
     let i = getInt(instr, state.env, 0)
-    if(i > 65535 || i < 0) {
+    if(i > 65535 || i < 0 || (55295 < i && i < 57344)) {
```

Comment:
> We could create a bug for supporting beyond-BMP characters in the JavaScript 

Does it mean making this range larger, even though chars are UTF-16 code points? Allow integers from 65535 to 1114111? (all the unicode scalar values)

Reply:
Good question. Yes, I suppose the focus of that proposed issue would be removing the `i > 65535` part of this condition: supporting larger values that map to a single Unicode character but multiple JavaScript characters (i.e., JavaScript strings with `len()` greater than 1).

For those values, it looks like [`fromCharCode(n)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode) won't work by itself, but it might be as easy as switching to [`fromCodePoint`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint), which would be awesome and convenient if it works!!

Reply:
I think I got it...